projects/vdk-plugins/vdk-jupyter/.github/workflows/binder-on-pr.yml

```diff
@@ -0,0 +1,30 @@
+# Reference https://mybinder.readthedocs.io/en/latest/howto/gh-actions-badges.html
```

Comment:
What are those github actions ?

Reply:
They were automatically generated by the template and should be removed or changed in the next PR.